extensions/audit/pkg/types/conversion.go

```diff
@@ -234,8 +234,8 @@ func FilesAuditEvent(base AuditEvent, itemid, owner, path string) AuditEventFile
 
 // ContainerCreated converts a ContainerCreated event to an AuditEventContainerCreated
 func ContainerCreated(ev events.ContainerCreated) AuditEventContainerCreated {
-	iid, path, uid := extractFileDetails(ev.Ref, ev.Executant)
-	base := BasicAuditEvent(uid, "", MessageContainerCreated(iid), ActionContainerCreated)
+	iid, path, uid := extractFileDetails(ev.Ref, ev.Owner)
+	base := BasicAuditEvent(uid, "", MessageContainerCreated(ev.Executant.OpaqueId, iid), ActionContainerCreated)
```

Comment:
Not critical. But better to use `ev.Executant.GetOpaqueId()` to avoid panic when executant is nil

Reply:
True, I'll change that. :+1: